src/engine/MaxConstraint.cpp

```diff
@@ -38,6 +38,8 @@ MaxConstraint::MaxConstraint( unsigned f, const Set<unsigned> &elements )
     , _maxIndexSet( false )
     , _maxLowerBound( FloatUtils::negativeInfinity() )
     , _obsolete( false )
+    , _eliminatedSomeVariables ( false ) // todo added
+    , _maxValueOfEliminated ( FloatUtils::negativeInfinity() ) // todo added
```

Comment:
remove space before (, remove comments

eliminatedSomeVariables --> remove the "some". Maybe eliminatedVariables, or haveEliminatedVariables, or variablesEliminated.